packages/core/src/events/interfaces.ts

```diff
@@ -26,7 +26,7 @@ export type Integrations = {
 }
 
 // renamed
-export type CoreOptions = {
+export interface CoreOptions {
```

Comment:
Just curious - was this change needed or just being consistent?

Reply:
You mean the rename? I thought it made sense to prefix the things that were going to become base classes / interface (e.g. in node, we extend these options with Identity). However, being totally consistent here is hard.

Reply:
Oh, I meant why change from `type` to `interface`, but was just curious.